src/debug/daccess/daccess.cpp

```diff
@@ -4607,7 +4653,12 @@ ClrDataAccess::TranslateExceptionRecordToNotification(
         case DACNotify::JIT_NOTIFICATION:
             notify->OnCodeGenerated(pubMethodInst);
             break;
-
+        case DACNotify::JIT_NOTIFICATION2:
```

Comment:
We don't need to handle JIT_NOTIFICATION above. However if our DAC using tool (sos, clrmd, windbg, other) doesn't implement IXCLRDataExceptionNotification5 we should issue the notify->OnCodeGenerated callback in response to JIT_NOTIFICATION2. This allows older versions of these tools to keep operating successfully on the newer runtime.